src/test/java/uk/gov/hmcts/reform/amapi/controllers/AccessManagementControllerUnitTest.java

```diff
@@ -86,7 +86,7 @@ public void testCreateResourceAccess() throws Exception {
 
         doNothing().when(accessManagementService).grantExplicitResourceAccess(any());
 
-        this.mvc.perform(post("/api/access-resource")
+        this.mvc.perform(post("/api/v1/access-resource")
```

Comment:
Please define a global "version" variable and replace all of the version hardcoding in this file

Reply:
done